cardano-node-chairman/test/Spec/Chairman/Chairman.hs

```diff
@@ -74,6 +74,5 @@ chairmanOver H.Conf {..} allNodes = do
   case chairmanResult of
     Right ExitSuccess -> return ()
     _ -> do
-      H.note_ $ "Failed with: " <> show chairmanResult
-      H.noteM_ $ H.noteTempFile logDir $ "chairman" <> ".stdout.log"
-      H.noteM_ $ H.noteTempFile logDir $ "chairman" <> ".stderr.log"
+      H.note_ $ "Chairman failed with: " <> show chairmanResult
+      H.failure
```

Comment:
The reason was that the an assertion at this point got dropped accidentally, probably during the refactoring that exposed functionality to launch testnets independently.